### PR TITLE
add app property when the app was launched from steam

### PR DESCRIPTION
**Pull request by ZappoMan**

### CODE REVIEW

**Comment by ZappoMan** [comment]
Test Plan:
- this should have no meaningful change in the application for now. we will use this setting later/soon

File: interface/src/main.cpp [hunk 1]
```diff
@@ -161,6 +165,9 @@ int main(int argc, const char* argv[]) {
         QSettings::setDefaultFormat(QSettings::IniFormat);
         Application app(argc, const_cast<char**>(argv), startupTime, runServer, serverContentPathOptionValue);
 
+        QVariant launchedFromSteamVariant(launchedFromSteam);
```
Comment (jherico) [inline interface/src/main.cpp]: you don't have to explicitly create a QVariant here.  QVariant has a `bool` constructor so you can just say `app.setProperty("com.highfidelity.launchedFromSteam", launchedFromSteam);`

**Comment by hifi-gustavo** [comment]
Build console output for commit: 5f2b4c2a7f24111980b819a7252b0b64d8502ea6 available at: https://gist.github.com/416ea885bb2c696be474f8bba395c7ec<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR8920-5f2b4c2.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR8920-5f2b4c2.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: e464858c4c4034cd6f5f7f78df414e18d62443f1 available at: https://gist.github.com/416ea885bb2c696be474f8bba395c7ec<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR8920-e464858.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR8920-e464858.dmg

**Pull request merged**